### DISCUSSION TITLE
Не воспроизводит видео с ЮтЮб в автомобиле на АндроидАвто/doesn't work video from YouTube on screen in my car

### DISCUSSION THREAD

**Discussion by SatterCane** [AndreyPavlenko/Fermata#420] [General]
Femata media service не воспроизводит видео с ютюб на дисплее в автомобиле. Идет только звук, а видео воспроизводится только на экране смартфона (samsung s21, android 14). 
Femata media service does not play YouTube videos on the display in the car. There is only sound, and the video is played only on the smartphone screen (samsung s21, android 14).

**Comment by AndreyPavlenko** [comment]
https://github.com/AndreyPavlenko/Fermata/issues/327#issuecomment-1934257166

**Comment by MICHAIL974** [comment]
AA Wireless подключено через приложение, а в фермато на ГУ нет видео тольо звук, нет кнопок ютюб и браузера.ЧТО делать подскажите?

**Reply by AndreyPavlenko** [reply]
Что значит "подключено через приложение"?

**Comment by AndreyPavlenko** [comment]
Femata media service и не должно воспроизводить. Нужен AAWireless с включенным dev mode - https://github.com/AndreyPavlenko/Fermata/discussions/432

**Reply by MICHAIL974** [reply]
Что с AAW, что без AAW, ферма 1.9.9(237) работает в AA ограниченно. Нет видео, только звук, нет кнопок YouTube и перехода в браузер. Вот что я имел в виду.

**Reply by AndreyPavlenko** [reply]
В настройках AAW включите dev mode.

**Reply by MICHAIL974** [reply]
Developer mode нет этого пункта

**Comment by AndreyPavlenko** [comment]
Есть. Подключитесь к устройству.

**Reply by MICHAIL974** [reply]
Андрей приветствую. Разработчика включил в AAW. Спасибо. А сейчас запрашивает какие то разрешения и открывает в телефоне специальные возможности, какие разрешения непонятно. 
![20250925_205840](https://github.com/user-attachments/assets/36f8c1d2-f73b-4d9e-b03d-7982faf4f503)
![Screenshot_20250925_215830_Settings (1)](https://github.com/user-attachments/assets/f8048b27-db98-46f3-ab41-9dded9f4e2a7)

**Reply by AndreyPavlenko** [reply]
Это вы зеркало запускаете, а не плеер. Если нужно именно оно, то нужно дать все запрашиваемые пермишины. Если нужна Fermata Auto, то там другая иконка, синяя.

**Reply by MICHAIL974** [reply]
Понял, спасибо большое.
